File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/Orders/ConfirmReservation.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.Orders
{
    public class ConfirmReservation : Command<Guid>
    {
        public bool IsReservationSuccess { get; set; }

        public ConfirmReservation() { }
        public ConfirmReservation(Guid orderId, bool isReservationSuccess) : base(orderId)
        {
            IsReservationSuccess = isReservationSuccess;
        }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/Orders/ConfirmReservation.java
package com.microsoft.conference.common.registration.commands.Orders;

import com.enodeframework.commanding.Command;

public class ConfirmReservation extends Command<String> {
    public boolean IsReservationSuccess;

    public ConfirmReservation() {
    }

    public ConfirmReservation(String orderId, boolean isReservationSuccess) {

        super(orderId);
        IsReservationSuccess = isReservationSuccess;
    }
}
